Argo/Types/JSON.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// A type safe representation of JSON.
 public enum JSON {
-  case object([String: JSON])
+  case object([AnyHashable: JSON])
```

Comment:
Ah, good point. I made this change because I was getting an EXC_BAD_ACCESS crash when trying to run the tests. I'll see if I can work around that a different way.